include_core/omrgcconsts.h

```diff
@@ -65,14 +65,12 @@ typedef enum MM_GCPolicy {
 #define OMR_GC_WRITE_BARRIER_TYPE_CARDMARK_INCREMENTAL 0x4
 #define OMR_GC_WRITE_BARRIER_TYPE_CARDMARK_AND_OLDCHECK 0x5
 
-/* barriers types to trigger both pre-store and post-store
- * (in theory, SATB_AND_CARDMARK_AND_OLDCHECK would too,
- * but not needed by any GC policy) */
-#define OMR_GC_WRITE_BARRIER_TYPE_ALWAYS 0x6
```

Comment:
It will be more clear why these changes are required once corresponding OpenJ9 PR is opened. We'll require some  intermediate changes on OpenJ9 side before this PR goes in.